src/PlaywrightSharp.Tests/Chromium/JSCoverageTests.cs

```diff
@@ -13,10 +12,14 @@ namespace PlaywrightSharp.Tests.Chromium
 {
     ///<playwright-file>chromium/chromium.spec.js</playwright-file>
     ///<playwright-describe>JSCoverage</playwright-describe>
-    public class CoverageTests : PlaywrightSharpPageBaseTest
+    [Trait("Category", "chromium")]
+    [Trait("Category", "firefox")]
```

Comment:
I think we should add firefox and webkit once we actually implement `ICoverage` for them

Reply:
That's a good idea, but on the other side, these tests are ignored for firefox and webkit. There are no coverage classes on firefox and webkit.

Reply:
for me it's a bit confusing since it seems that these tests are covered in firefox & webkit. maybe these tests should only have the chromium trait

Reply:
agreed